build.sbt

```diff
@@ -491,19 +492,37 @@ lazy val `persistence-cassandra` = (project in file("persistence-cassandra"))
     )
   ) configs (MultiJvm)
 
-lazy val `persistence-jdbc` = (project in file("persistence-jdbc"))
-  .settings(name := "lagom-javadsl-persistence-jdbc")
-  .dependsOn(persistence % "compile;test->test", `persistence-javadsl` % "compile;test->test")
+
+lazy val `persistence-jdbc-core` = (project in file("persistence-jdbc/core"))
```

Comment:
currently core only contains the SlickProvider, all other classes was specific to the JavaDSL